apps/frontend/src/pages/project/project.tsx

```diff
@@ -31,12 +31,25 @@ const Project = (): JSX.Element => {
 
 	const isRejected = projectStatus === DataStatus.REJECTED;
 
+	const hasProject = project !== null;
```

Comment:
to keep it consistent with other places
```suggestion
	const hasProject = Boolean(project);
```

Reply:
yeah i tried it first, but the problem that `'project' is possibly 'null'` appears again